### PR TITLE
Batch modify file extension

### PR DESCRIPTION
**Pull request by LokHsu**
## Description

+ new script command:
+ commands/system/modify-extension.sh:
+ Batch modify the file in the current directory or the specified extension of the selected file
## Type of change

- [x] New script command

## Screenshot

GIF1

GIF2

## Dependencies / Requirements

+ You must enter the search extension in 'Before' and the modified extension in 'After'

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/modify-extension.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Modify extension
```
Comment (unnamedd) [inline commands/system/modify-extension.sh:5]:
```suggestion
# @raycast.title Modify File Extension
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @LokHsu, a really small change to be done here to keep your `title` in conformance to the repository’s guideline.

**Comment by LokHsu** [comment]
> @LokHsu，您好，这里要做的很小的更改是为了使您`title`与存储库的指南保持一致。

I have finished the change of this file according to your suggestion.

**Comment by dehesa** [comment]
Pretty useful stuff, thanks @LokHsu

**Pull request merged**